### DISCUSSION TITLE
no root user

### DISCUSSION THREAD

**Discussion by elvenaz** [SpiralLinux/SpiralLinux-project#43] [General]
hi, im just install the iso and I am trying to enter root with the su command but it says that the password does not match, and when reviewing the installation at no time is it created or asked for root creation, how can I create the root user as admin or what is the solution for this.

Cheers

**Comment by geckolinux** [comment]
Hi there, yes, SpiralLinux follows the Ubuntu style of disabling the root account by default. To enable it:
`sudo passwd root`
